test/images/agnhost/netexec/netexec.go

```diff
@@ -693,7 +693,11 @@ func startSCTPServer(sctpPort int) {
 	for {
 		conn, err := listener.AcceptSCTP()
 		assertNoError(err, fmt.Sprintf("failed accepting SCTP connections"))
-		clientAddress := conn.RemoteAddr().String()
+		remoteAddr, err := conn.SCTPRemoteAddr(0)
```

Comment:
this seems the same as `conn.RemoteAddr()`, no?

Reply:
Yes, the underlying call is the same. It additionally returns an error on which we assert and exit. 

https://github.com/kubernetes/kubernetes/pull/124425#issuecomment-2068038859